### PR TITLE
Styling and grammar fixes

### PR DESCRIPTION
**Pull request by ab3lson**
## Proposed change
The capitalization of Spotify was fixed and the activation process was made more thorough.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/spotify.markdown [hunk 1]
```diff
@@ -12,25 +12,25 @@ ha_codeowners:
 ha_domain: spotify
 ---
 
-The `spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
+The `Spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
The Spotify media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
```

File: source/_integrations/spotify.markdown [hunk 2]
```diff
@@ -12,25 +12,25 @@ ha_codeowners:
...
 
 ## Prerequisites
 
-- Spotify account
-- Spotify application, properly configured (see below).
+- `Spotify` account
+- `Spotify` application configured for Home Assistant (see [below](#create-a-spotify-application))
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
- Spotify account
- Spotify application configured for Home Assistant (see [below](#create-a-spotify-application))
```

File: source/_integrations/spotify.markdown [hunk 3]
```diff
@@ -12,25 +12,25 @@ ha_codeowners:
...
 
 <div class='note'>
-  Controlling the Spotify integration (pause, play, next, etc.) requires a Premium account.
-  If you do not have a Premium account, the integration in the frontend will not show the controls.
+  Spotify integrated media controls (play, pause, skip) requires a Premium account.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]: It is not limited to media control or play, pause and skip. This change now made it untrue.

I suggest restoring the first part of this sentence.

File: source/_integrations/spotify.markdown [hunk 4]
```diff
@@ -12,25 +12,25 @@ ha_codeowners:
...
 
-To create the required Spotify application:
+### Create a Spotify application
 
 - Login to [Spotify Developer](https://developer.spotify.com).
 - Visit the [My Applications](https://developer.spotify.com/my-applications/#!/applications) page.
 - Select **Create An App**. Enter any name and description.
-- Once your application is created, view it and copy your **Client ID** and **Client Secret**, which are used in the Home Assistant configuration file.
-- On the Spotify page, enter the **Edit Settings** dialog of your newly created application and add a **Redirect URI**:
+- Once your application is created, view it and copy your `Client ID` and `Client Secret`, which are used in the Home Assistant configuration file.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]: Please restore, Client ID & Secret to not refer to code and should not be code fenced.

File: source/_integrations/spotify.markdown [hunk 5]
```diff
@@ -41,7 +41,7 @@ To create the required Spotify application:
 
 ## Configuration
 
-To add Spotify to your installation, add the following to your `configuration.yaml` file:
+To add `Spotify` to Home Assistant, add the following to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
To add Spotify to Home Assistant, add the following to your `configuration.yaml` file:
```

File: source/_integrations/spotify.markdown [hunk 6]
```diff
@@ -63,12 +63,11 @@ client_secret:
...
 
-After you have set up the above:
-
 - Go to the **Configurations** page in the Home Assistant frontend.
-- Go to **Integrations**.
-- Add a new **Spotify** integration.
-- Follow the steps shown to authenticate Home Assistant with your Spotify account.
+- Click on **Integrations**.
+- Click the orange **+** on the bottom-right.
+- Click on `Spotify`.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
- Click on "Spotify".
```

File: source/_integrations/spotify.markdown [hunk 7]
```diff
@@ -63,12 +63,11 @@ client_secret:
...
+- Once the new `Spotify` tab opens, enter your `Spotify` credentials and allow Home Assistant to access `Spotify`.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
- Once the new Spotify tab opens, enter your Spotify credentials and allow Home Assistant to access your Spotify account.
```

File: source/_integrations/spotify.markdown [hunk 8]
```diff
@@ -81,22 +80,22 @@ After you have set up the above:
...
 ## Using multiple Spotify accounts
 
-The integration support multiple Spotify accounts at once. You don't need to
-create another Spotify Application in the Spotify Developer portal and no
-modification to the `configuration.yaml` file is needed. Multiple Spotify
-accounts can be linked on a _single_ Spotify application.
+The integration support multiple `Spotify` accounts at once. You don't need to
+create another `Spotify` application in the `Spotify` Developer Portal and no
+modification to the `configuration.yaml` file is needed. Multiple `Spotify`
+accounts can be linked to a _single_ `Spotify` application.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
The integration support multiple Spotify accounts at once. You don't need to
create another Spotify application in the Spotify Developer Portal and no
modification to the `configuration.yaml` file is needed. Multiple Spotify
accounts can be linked to a _single_ Spotify application.
```

File: source/_integrations/spotify.markdown [hunk 9]
```diff
@@ -81,22 +80,22 @@ After you have set up the above:
...
 To add an additional Spotify account to Home Assistant, repeat _only_ the steps
 in the [Activating the Spotify integration](#activating-the-spotify-integration)
-chapter. However, make sure you go to the Spotify website and log out before
+section. However, make sure you go to the `Spotify` website and log out before
 you do so.
 
 ## URI Links for Playlists
 
-You can send playlists to Spotify using the `"media_content_type": "playlist"`, which are part of the
+You can send playlists to `Spotify` using the `"media_content_type": "playlist"`, which is part of the
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
You can send playlists to Spotify using the `"media_content_type": "playlist"`, which is part of the
```

File: source/_integrations/spotify.markdown [hunk 10]
```diff
@@ -81,22 +80,22 @@ After you have set up the above:
...
-accounts can be linked on a _single_ Spotify application.
+The integration support multiple `Spotify` accounts at once. You don't need to
+create another `Spotify` application in the `Spotify` Developer Portal and no
+modification to the `configuration.yaml` file is needed. Multiple `Spotify`
+accounts can be linked to a _single_ `Spotify` application.
 
 To add an additional Spotify account to Home Assistant, repeat _only_ the steps
 in the [Activating the Spotify integration](#activating-the-spotify-integration)
-chapter. However, make sure you go to the Spotify website and log out before
+section. However, make sure you go to the `Spotify` website and log out before
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
section. However, make sure you go to the Spotify website and log out before
```

File: source/_integrations/spotify.markdown [hunk 11]
```diff
@@ -81,22 +80,22 @@ After you have set up the above:
...
 you do so.
 
 ## URI Links for Playlists
 
-You can send playlists to Spotify using the `"media_content_type": "playlist"`, which are part of the
+You can send playlists to `Spotify` using the `"media_content_type": "playlist"`, which is part of the
 [media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
-The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+The `media_content_id` can be obtained from the `Spotify` desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the `Spotify` phone and web app).
```
Comment (frenck) [inline source/_integrations/spotify.markdown]:
```suggestion
The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).

```

File: source/_integrations/spotify.markdown [hunk 12]
```diff
@@ -81,22 +80,22 @@ After you have set up the above:
...
 
 ## Unsupported Devices
 
-- **Sonos**: Although Sonos is a Spotify Connect device, it is not supported by the official Spotify API.
+- **Sonos**: Although Sonos is a `Spotify Connect` device, it is not supported by the official `Spotify` API.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]: Revert this change

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @ab3lson 👍 

However, we don't code fence brand names.

File: source/_integrations/spotify.markdown (hunk 3 above)
Comment (ab3lson) [reply source/_integrations/spotify.markdown]: Reverted.

File: source/_integrations/spotify.markdown (hunk 4 above)
Comment (ab3lson) [reply source/_integrations/spotify.markdown]: Sorry. I am new to this. I'm learning what should and should not be code fenced

File: source/_integrations/spotify.markdown (hunk 12 above)
Comment (ab3lson) [reply source/_integrations/spotify.markdown]: Reverted.

**Review by ab3lson — COMMENTED** [review]
I tried to make fixes according to all your comments.

**Review by frenck — APPROVED** [review]
Thanks, @ab3lson 👍 

Happy Hacktoberfest 🎉
